Formula/mariadb-connector-odbc.rb

```diff
@@ -0,0 +1,66 @@
+class MariadbConnectorOdbc < Formula
+  desc "MariaDB Standardized database driver"
+  homepage "https://downloads.mariadb.org/connector-odbc/"
+  # MariaDB's download page doesn't let you download without going through the site, probably a cookie-based check
+  # The connector is packaged at http://archive.mariadb.org/connector-odbc-2.0.13/source/mariadb-connector-odbc-2.0.13-ga-src.tar.gz, but there's no https here
+  # So the next best choice seems to be github:
+  url "https://github.com/MariaDB/mariadb-connector-odbc/archive/2.0.13.tar.gz"
+  sha256 "6b77e879335ea32dad40d42de95930df8ad557faa887b414a09ab7ead3ce963c"
+
+  depends_on "cmake" => :build
+  depends_on "mariadb-connector-c"
+  depends_on "unixodbc"
+  depends_on "openssl"
+
+  patch :DATA
```

Comment:
@fxcoudert unfortunately the patch I've sent them doesn't apply to the 2.0.13.tar.gz I'm downloading as base package. 

The patch as submitted includes an update to .gitingore but that file isn't in the 2.0.13.tar.gz due to https://github.com/MariaDB/mariadb-connector-odbc/blob/master/.gitattributes - so the patch fails with a 'No file to patch'

Not sure how to work around that.